docs-mslearn/toolkit/help/errors.md

```diff
@@ -714,6 +714,28 @@ Applicable versions: **0.1 - 0.1.1** (fixed in **0.2**)
 
 <br>
 
+## Power BI: We cannot convert the value null to type Table
+
+<sup>Severity: Critical</sup>
+
+This error most commonly occurs when connecting Power BI storage reports to a storage account using the Blob endpoint instead of the required Data Lake Storage (DFS) endpoint.
```

Comment:
Is this the only case where this error can happen? I'm guessing there might be multiple reasons. Should we say that one reason might be this?